lib/clean-host.js

```diff
@@ -26,7 +26,7 @@ function rtrim(value) {
 module.exports = function cleanHostValue(value) {
   value = String(value).trim().toLowerCase();
 
-  var parts = URL.parse(hasPrefixRE.test(value) ? value : '//' + value, null, true);
```

Comment:
Do you remember why this `'//' + value` was introduced in the first place? I thought it was serving this purpose already.

Reply:
Can't remember but it makes no difference for `require('url').parse` (still a pathname, no host detected).